src/core/getDebugFrameRequest.ts

```diff
@@ -0,0 +1,50 @@
+import { FrameRequest, FrameValidationData } from './types';
+
+export type DebugFrameRequestOptions = {
+  following?: boolean; // Indicates if the viewer clicking the frame follows the cast author
+  interactor?: {
+    fid?: number; // Viewer Farcaster ID
+    custody_address?: string; // Viewer custody address
+    verified_accounts?: string[]; // Viewer account addresses
+  };
+  liked?: boolean; // Indicates if the viewer clicking the frame liked the cast
+  recasted?: boolean; // Indicates if the viewer clicking the frame recasted the cast
+};
+
+type DebugFrameRequest = FrameRequest & { onchainkitDebug: Required<FrameValidationData> };
+
+/**
+ * Modify a standard frame request to include simulated values (e.g., indicate the viewer
+ * follows the cast author) for development/debugging purposes.
+ * @param request A standard frame request.
+ * @param options An object containing values we will pretend are real for the purposes of debugging.
+ * @returns
+ */
+function getDebugFrameRequest(
```

Comment:
Here me out, what if this is actually called `getMockFrameRequest`, I think can be very powerful also to write tests.

I see the word Mock feel more like utilities tests vibes.